### PR TITLE
create picasso

### PR DESCRIPTION
**Pull request by cameroncuster**

### CODE REVIEW

File: picasso/problem.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  "problem": "Given the amount of money Tess and Maxie will bid on each painting, as well as how many paintings Maxie will buy, determine the maximum amount of money Katie can earn if she sells her paintings optimally.",
+  "input": "The first line of the input will contain a single integer, $a$, representing the number of auctions. Each auction will be represented by multiple lines. The first line of each auction will contain two integers, $n$ and $x$ $(1 \\leq x \\leq n \\leq 1,000)$, representing the number of paintings Katie has to sell, and the number of paintings Maxie will buy, respectively. The next line will contain $n$ integers, $0 \\leq t_1, t_2, ..., t_n \\leq 10^5$ , where $t_i$ is the amount of money Tess will pay for the $i{th}$ painting. The next line will contain $n$ integers, $m_1, m_2, ..., m_n$ (each between 0 and $10^5$, inclusive), where $m_i$ is the amount of money Maxie will pay for the $i{th}$ painting.",
```
Comment (tylerm390) [inline picasso/problem.json]: Is \\$i{th}\\$ supposed to be \\$i^{th}\\$ for the superscript?

File: picasso/data/picasso.in
Comment (tylerm390) [inline picasso/data/picasso.in:1]: Maybe we should have an addition case for samples like Kattis?

File: picasso/problem.json (hunk 1 above)
Comment (cameroncuster) [reply picasso/problem.json]: oh yeah I think so, good catch

**Review by tylerm390 — APPROVED** [review]
lgtm

**Pull request merged**